src/components/multilineInput/SplitMultilineInput.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from 'react';
+import * as _ from 'underscore';
+
+import {Input} from '../input/Input';
+import {Label} from '../input/Label';
+import {AddInputAction} from './AddInputAction';
+import {DeleteInputAction} from './DeleteInputAction';
+import * as styles from './styles/SplitMultilineInput.css';
+
+export interface ISplitInput {
+    id: string;
+    label: string;
+    placeholder?: string;
+    validation?: (value: any) => boolean;
+    validationMessage?: string;
+}
+
+export interface ISplitValue {
+    [inputId: string]: any;
+}
+
+export interface ISplitMultilineInputOwnProps {
+    inputs: ISplitInput[];
+    defaultValues: ISplitValue[];
+    onChange?: (values: ISplitValue[]) => void;
+}
+
+export interface ISplitMultilineInputProps extends ISplitMultilineInputOwnProps {}
+
+export interface ISplitMultilineInputState {
+    values: ISplitValue[];
+}
+
+export class SplitMultilineInput extends React.PureComponent<ISplitMultilineInputProps, ISplitMultilineInputState> {
+
+    constructor(props: ISplitMultilineInputProps, state: ISplitMultilineInputState) {
+        super(props, state);
+        this.state = {
+            values: this.props.defaultValues,
+        };
+    }
+
+    componentDidUpdate(prevProps: ISplitMultilineInputProps) {
+        if (!_.isEqual(prevProps.defaultValues, this.props.defaultValues)) {
+            this.setState({
+                values: this.props.defaultValues,
+            });
+        }
+    }
+
+    render() {
+        return (
+            <div className={styles.inputBlock}>
+                <ul>
+                    {this.getDeletableInputs()}
+                    {this.getNewInput()}
+                </ul>
+            </div>
+        );
+    }
+
+    private getDeletableInputs() {
+        return _.map(this.state.values, (values: ISplitValue, index: number) => {
+            const labelIds: string[] = _.keys(values);
+            const inputLength: number = labelIds.length;
+            const inputs: JSX.Element[] = _.map(labelIds, (labelId: string, inputIndex: number) => {
+                const input: ISplitInput = _.findWhere(this.props.inputs, {id: labelId});
+
+                if (input) {
+                    const deleteButton: JSX.Element = inputIndex + 1 === inputLength ? <DeleteInputAction onClick={() => this.removeLine(index)} /> : null;
+                    return (
+                        <Input
+                            classes={styles.input}
+                            value={values[labelId]}
+                            placeholder={input.placeholder}
+                            validate={input.validation ? (value: any) => input.validation(value) : undefined}
+                            key={labelId + inputIndex}
+                            onChange={(value?: string, valid?: boolean) => this.changeValue(value, valid, index, labelId)}
+                        >

```

Comment:
pretty much the same but you have the `labelTitle` and `labelProps` directly on the Input component too, you could create your label from there, but it's not major